main.js

```diff
@@ -32,9 +32,20 @@ var pathToParamedicConfig = argv.config && path.resolve(argv.config);
 if (pathToParamedicConfig || // --config
     argv.platform && argv.plugin) { // or --platform and --plugin
 
-    var paramedicConfig = pathToParamedicConfig ?
-        ParamedicConfig.parseFromFile(pathToParamedicConfig):
-        ParamedicConfig.parseFromArguments(argv);
+    var paramedicConfig;
+    if(pathToParamedicConfig) {
+        paramedicConfig = ParamedicConfig.parseFromFile(pathToParamedicConfig);
+
+        if(argv.plugin) {
+            paramedicConfig.setPlugins(argv.plugin);
```

Comment:
Design comment: instead of having different mutually incompatible parsing methods for `ParamedicConfig`, its constructor can instead take a JSON object, which in turn can be a combination of args options and the config file.

Outside of the design comment though, this logic can be simplified to just set the properties from `argv` after calling `parseFromFile`.